contracts/RelayHub.sol

```diff
@@ -194,6 +196,7 @@ contract RelayHub is IRelayHub {
         EnvelopingTypes.RelayRequest calldata relayRequest,
         bytes calldata signature
     ) external override returns (bool destinationCallSuccess){
+        (signature);
```

Comment:
i know this was commented on the original PR, but i am baffled by it. do we have any idea why this statement reduces costs?

i don't even know how a programmer would even arrive at this :exploding_head:

Reply:
let's leave the spooky statement in, but maybe add an in-line comment to explain

Reply:
```suggestion
        /* statement originally present in the GSN repo; 
         * we left it here because of a small gas improvement noticed on tests execution.
         */
        (signature);
```